list/internal/listitem/_list-item.scss

```diff
@@ -134,6 +134,7 @@
     display: inline-flex;
     justify-content: center;
     flex-direction: column;
+    max-width: 100%;
```

Comment:
`max-width: 100%` will make the text overflow if there's also start or end content. This is because the `.body` is trying to take up the whole width of the flex container.

Change this to `min-width: 0` and watch the magic happen.

See https://stackoverflow.com/a/66689926 for a more in-depth explanation as to why this works.

Reply:
@asyncLiz Oh, of course. I just updated the PR. Thanks for the review!